Page/EducationPage.cs

```diff
@@ -0,0 +1,195 @@
+                          // string expectedText = "Education has been added";
+
+                           if (languageRead.Text == expectedText)
+                           {
+                                 test.Log(Status.Info, expectedText);
+                               //test.Log(Status.Info, "Add Education Passed");
+
+                }
+                           else
+                           {
+                               test.Log(Status.Fail, "Add Education failed");
+                               // Capture and attach the screenshot
+                               string screenshotPath = CaptureScreenshot(TestContext.CurrentContext.Test.Name);
+                               test.AddScreenCaptureFromPath(screenshotPath);
+                           }
+                       }
+                       catch (Exception ex)
+                       {
+                           // Log the failure
+                           test.Log(Status.Fail, "Test failed: " + ex.Message);
+
+                           // Capture and attach the screenshot
+                           string screenshotPath = CaptureScreenshot(TestContext.CurrentContext.Test.Name);
+                           test.AddScreenCaptureFromPath(screenshotPath);
+                           throw;
+                       } 
+
+                   }
+       
+        private static string CaptureScreenshot(string testName)
+                   {
+                       try
+                       {
+                           string screenshotDir = Path.Combine(Directory.GetCurrentDirectory(), "Screenshots");
+                           Directory.CreateDirectory(screenshotDir); // Ensure the directory exists
+
+                           string screenshotPath = Path.Combine(screenshotDir, $"{testName}_{DateTime.Now:yyyyMMdd_HHmmss}.png");
+                           Screenshot screenshot = ((ITakesScreenshot)driver).GetScreenshot();
+                           screenshot.SaveAsFile(screenshotPath, ScreenshotImageFormat.Png);
+
+                           return screenshotPath;
+                       }
+                       catch (Exception ex)
+                       {
+                           test.Log(Status.Error, "Failed to capture screenshot: " + ex.Message);
+                           return null;
+                       }
+                   }
+
+        //Locating and clicking the Level dropdown
+        public static void Country(string country)
+        {
+            IWebElement levelDropdown = driver.FindElement(By.XPath("//*/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[1]/div[2]/select"));
+            levelDropdown.SendKeys(country);
+
+
+        }
+        public static void LocateEnterDegreeTextbox(string degree)
+        {
+            IWebElement languageTextbox = driver.FindElement(By.XPath("//*/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[2]/div[2]/input"));
```

Comment:
you need to use the POM pattern your web elements should be reusable via instance variables